pkg/cloud/azure/actuators/machine/reconciler.go

```diff
@@ -283,9 +298,44 @@ func (s *Reconciler) Update(ctx context.Context) error {
 		Message: machineCreationSucceedMessage,
 	})
 
+	s.setPrettyMachineAnnotations(vm)
```

Comment:
can we not tie func name to one consumer, e.g just `setMachineAnnotations`?

Reply:
There are other annotations that are set outside of this method

Reply:
Also since this is purely informative can we use labels instead, similar to what the node does ```  labels:
    beta.kubernetes.io/arch: amd64
    beta.kubernetes.io/instance-type: Standard_D2s_v3
    beta.kubernetes.io/os: linux
    failure-domain.beta.kubernetes.io/region: centralus
    failure-domain.beta.kubernetes.io/zone: centralus-1
    kubernetes.io/arch: amd64
    kubernetes.io/hostname: agl230719-bl7np-worker-centralus1-nqvkw
    kubernetes.io/os: linux
    node-role.kubernetes.io/worker: ""
    node.openshift.io/os_id: rhcos```

Reply:
Method renamed to `setMachineCloudProviderSpecifics`